### PR TITLE
5.3 Release

### PR DESCRIPTION
**Pull request by secureideas**
This is the pull request to make SamuraiWTF 5.3 the release.   We will need to tag the files after accepting the PR.

### CODE REVIEW

File: amazon-linux/Vagrantfile [hunk 1]
```diff
@@ -63,7 +63,7 @@ Vagrant.configure("2") do |config|
     samuraiwtf.vm.provider "virtualbox" do |vb|
     # Display the VirtualBox GUI when booting the machine
       vb.gui = true
-      vb.name = "SamuraiWTF-5.2"
+      vb.name = "SamuraiWTF-6.0"
```
Comment (mgillam) [inline amazon-linux/Vagrantfile:66]: Is there a reason this wouldn't be the same version as the `Vagrantfile` at the root?

**Review by mgillam — DISMISSED** [review]
One question where `amazon-linux/Vagrantfile` has a different version (6.0) from the rest (5.3). Otherwise all looks fine.

File: amazon-linux/Vagrantfile (hunk 1 above)
Comment (JGillam) [reply amazon-linux/Vagrantfile:66]: should be 5.3 I think

**Review by JGillam — CHANGES_REQUESTED** [review]
the one place we say 6.0 instead of 5.3 needs to be made consistent with the other changes before this is merged.

**Review by mgillam — APPROVED** [review]

**Pull request merged**